### PR TITLE
Adding rsyslog's pstats and more systemd fields

### PR DESCRIPTION
**Pull request by t0ffel**

### CODE REVIEW

File: namespaces/systemd.yml [hunk 1]
```diff
@@ -151,6 +151,71 @@ namespace:
...
+    - name: AUDIT_ID
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_TYPE
+      type: string
+      description: >
+        TODO
+
```
Comment (richm) [inline namespaces/systemd.yml:218]: Are these related to audit system logging?  If so, are they related to aushape?  @spbnick

File: namespaces/systemd.yml [hunk 2]
```diff
@@ -194,3 +259,108 @@ namespace:
...
+    - name: USER_UNIT
+      type: string
+      description: >
+        TODO
+
+    - name: SOURCE_MONOTONIC_TIMESTAMP
+      type: string
```
Comment (richm) [inline namespaces/systemd.yml:364]: This field and many others are type `string` - should they be `int` or `double` or some other numeric field type?

File: namespaces/systemd.yml (hunk 1 above)
Comment (t0ffel) [reply namespaces/systemd.yml:218]: I think these is when audit gets logged into journal.
We should probably move these to separate audit space at some point. I'm adding them for now, because they were hanging out there previously on the top level and we're working on reindexing the entire production ES cluster, so i want to keep track of these.

**Comment by richm** [comment]
@t0ffel any update on this PR?

**Comment by lukas-vlcek** [comment]
@spbnick @t0ffel Any update on this? Shall we close it for now? (Also I am not sure we should go for field names having exclamation mark in them).

**Comment by lukas-vlcek** [comment]
Shall we close this issue?
